core/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -74,59 +74,91 @@ object StaticFile {
       }
     })
 
+  def calcETag[F[_]: Sync]: File => F[String] = f => Sync[F].delay(f.length().toString)
```

Comment:
Is this a sensible default?  I tend to think of hashes of the content for accuracy, or maybe a combination of the timestamp and length for speed.